requirements.txt

```diff
@@ -28,3 +28,4 @@ s3fs==2022.5.0
 pandas
 selenium
 https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.1.0/en_core_web_sm-3.1.0.tar.gz
+sentence-transformers
```

Comment:
- Gotta change` webdriver-manager` to `webdriver_manager` 
- Need to specify `selenium==4.2.0` to avoid the `AttributeError: 'WebDriver' object has no attribute 'find_element_by_id'` error